src/test/resources/regressions/issues/000124.gobra

```diff
@@ -0,0 +1,11 @@
+// Any copyright is dedicated to the Public Domain.
+// http://creativecommons.org/publicdomain/zero/1.0/
+
+package main
+
+// ##(-I src/test/resources/regressions/issues/issue124/)
```

Comment:
What do you think? Would `000124` be a better folder name? This way, the folder should appear next to `000124.gobra`

Reply:
Yes, that is a better name indeed!